config/sysinfo/fail_commnds

```diff
@@ -0,0 +1,2 @@
+mkdir -p results && sosreport --batch --tmp-dir results
```

Comment:
filename is wrong, probably typo,
`fail_commnds` -> `fail_commands`
@narasimhan-v i wonder how it worked for you, probably you have that file named correctly in your local setup?

Reply:
Sorry, that was a typo, and had it working in my test env.

Fixed it now.